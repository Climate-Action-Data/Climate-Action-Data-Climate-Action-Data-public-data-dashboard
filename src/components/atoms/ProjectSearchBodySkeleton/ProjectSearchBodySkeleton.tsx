import { Tr, Td, Skeleton } from '@chakra-ui/react'

export const ProjectSearchBodySkeleton = () => {
  return (
    <Tr height="92px">
      <Td>
        <Skeleton height="20px" />
      </Td>
      <Td>
        <Skeleton height="20px" />
      </Td>
      <Td>
        <Skeleton height="20px" />
      </Td>
      <Td>
        <Skeleton height="20px" />
      </Td>
      <Td>
        <Skeleton height="20px" />
      </Td>
      <Td>
        <Skeleton height="20px" />
      </Td>
      <Td isNumeric>
        <Skeleton height="20px" />
      </Td>
      <Td isNumeric>
        <Skeleton height="20px" />
      </Td>
      <Td isNumeric>
        <Skeleton height="20px" />
      </Td>
      <Td isNumeric>
        <Skeleton height="20px" />
      </Td>
    </Tr>
  )
}
