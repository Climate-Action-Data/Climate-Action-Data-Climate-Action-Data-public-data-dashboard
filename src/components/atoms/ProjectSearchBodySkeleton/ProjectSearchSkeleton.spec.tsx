import { render } from '@testing-library/react'
import { Tbody, Table } from '@chakra-ui/react'

import { ProjectSearchBodySkeleton } from './ProjectSearchBodySkeleton'

it(`renders correctly`, () => {
  const { container } = render(
    <Table>
      <Tbody>
        <ProjectSearchBodySkeleton />
      </Tbody>
    </Table>,
  )
  expect(container).toMatchSnapshot()
})
