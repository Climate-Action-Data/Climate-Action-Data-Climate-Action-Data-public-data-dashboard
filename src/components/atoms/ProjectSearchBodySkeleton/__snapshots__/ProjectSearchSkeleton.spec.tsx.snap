// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <table
    class="chakra-table css-0"
  >
    <tbody
      class="css-0"
    >
      <tr
        class="css-mii0oe"
      >
        <td
          class="css-0"
        >
          <div
            class="chakra-skeleton css-1tjwkx3"
          />
        </td>
        <td
          class="css-0"
        >
          <div
            class="chakra-skeleton css-1tjwkx3"
          />
        </td>
        <td
          class="css-0"
        >
          <div
            class="chakra-skeleton css-1tjwkx3"
          />
        </td>
        <td
          class="css-0"
        >
          <div
            class="chakra-skeleton css-1tjwkx3"
          />
        </td>
        <td
          class="css-0"
        >
          <div
            class="chakra-skeleton css-1tjwkx3"
          />
        </td>
        <td
          class="css-0"
        >
          <div
            class="chakra-skeleton css-1tjwkx3"
          />
        </td>
        <td
          class="css-0"
          data-is-numeric="true"
        >
          <div
            class="chakra-skeleton css-1tjwkx3"
          />
        </td>
        <td
          class="css-0"
          data-is-numeric="true"
        >
          <div
            class="chakra-skeleton css-1tjwkx3"
          />
        </td>
        <td
          class="css-0"
          data-is-numeric="true"
        >
          <div
            class="chakra-skeleton css-1tjwkx3"
          />
        </td>
        <td
          class="css-0"
          data-is-numeric="true"
        >
          <div
            class="chakra-skeleton css-1tjwkx3"
          />
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
